### ISSUE TITLE
Discussion on creating demos for algorithms implemented in ImageBinarization.jl

### ISSUE THREAD

**Issue by ashwanirathee** [zygmuntszpak/ImageBinarization.jl#73]
In [#164](https://github.com/JuliaImages/juliaimages.github.io/issues/164),It was pointed out that even though ImageBinarization.jl has very detailed explanations in docstring.There is a need for setting up a demo page and move the explanations to the form of demos for each algorithm.

**Comment by johnnychen94** [comment]
A quite long time ago, I had a discussion with @zygmuntszpak on slack and I believe we have reached a consensus to move the detailed explanations as demo cards and add URL links back to the docstrings. I never made it because at that time DemoCards is at a very early stage. The main issues of the current docstrings IMO are:

* the docstrings are too long to quickly get an idea of how an algorithm should be used,
* latex strings are not well-formatted in REPL

I'm working on #70. It's almost done; just needs confirmation from @zygmuntszpak I'll set up DemoCards after that, and we can proceed with this.

Another issue related to this is https://github.com/zygmuntszpak/HistogramThresholding.jl/issues/34, that we need to refactor the codes to really get useful docstrings with `?Yen`. After #70, `?Yen` gives nothing.

Personally, I would recommend you to try zygmuntszpak/HistogramThresholding.jl#34 out since that gives some real challenges to modify the codebase without breaking anything. I believe you can learn more by doing that than writing demos.

**Comment by ashwanirathee** [comment]
It would be really exciting to do a medium size project(refactor task is fairly long) after I am used to JuliaImages.I think for the next fortnight,I'll do some 4-5 demos and then get on with helping on refactoring https://github.com/zygmuntszpak/HistogramThresholding.jl/issues/34

**Comment by zygmuntszpak** [comment]
Thanks, Johnny and Ashwani for working on this. As Johnny stated, the vision is to move the detailed LaTeX explanations as demo cards and add and URL to the demo in the docstring.
